tests/APIv0/SmokeTest.php

```diff
@@ -418,10 +543,13 @@ public function testDiscussionAddBookMark() {
      * @large
      */
     public function testRemoveDiscussionBookMark() {
+        $this->setSpamCount(10);
```

Comment:
What about doing this in `beforeSetupClass` or something, and setting it to a much higher number, so flood control would effectively be disabled for all tests in this class?

Reply:
i can move this to the BaseTest,  i'll set it to 100.